vocabularies/mining-survey-status.ttl

```diff
@@ -0,0 +1,63 @@
+﻿@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> ..
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix sstatus: <http://linked.data.gov.au/def/mining-survey-status/> 
+
+<http://linked.data.gov.au/def/mining-survey-status> a owl:Ontology .
+
+sstatus:conceptScheme a skos:ConceptScheme;
+  dct:created "2019-04-09T03:31:01.023Z"^^xsd:dateTime;
+  dct:modified "2019-04-09T03:34:27.949Z"^^xsd:dateTime;
+  skos:definition "The stage in the lifecycle of an exploration survey or activity (including research activities)"@en;
+  skos:prefLabel "Mining Survey or Activity Status"@en .
```

Comment:
What are some examples of "Mining Activity"? For me, "Activity" is too broad a term.  
Survey: a general view, examination, or description of someone or something.  
Activity: a thing that a person or group does or has done

Reply:
To me, 'Mining Survey' is too narrow, and, more importantly, too misleading. In my mind, it conjures up someone doing some sort of survey at a mine—surely, this is not what's intended?

Yes, activity is broad, and this is why I prefer the term, as the vocab can then encompass any thing geological, whether it's to do with mining, petroleum, CSG or some researcher working on the mineralisation of cyanophytes in the Proterozoic of NW Queensland, or, indeed, just studying these bacteria from morphological, nomenclatural, environmental and/or evolutionary points of view (i.e. an academic study).

Otherwise, we will end up with a plethora of overlapping and thus somewhat useless vocabs!!

After I ascertained the function of the vocab, I proposed the definition: "The stage in the lifecycle of an exploration survey or activity (including research activities)". I gave this definition to overcome the limitations of the terms 'Mining' and 'Survey', and, David, it was based on your original definition.

Essentially, I see this vocab as an activity tracker, whether it's to do with mining, mapping, an exploration project  or otherwise, but to embrace all things/work geological. Thus, I maintain we should also move away from the misleading term 'Mining'. Perhaps: 'Geological activity status' and focus on the attendant definition?